app/main/templates/reports.html

```diff
@@ -6,8 +6,10 @@
 
 {% block app_content %}
 
-    <div class="row">
-        <h3 class="text-center pt-2">Available reports for all Series</h3></br>
+    <div class="container py-5">
+        <div class="row">
+            <h1 class="h1 text-center justify-content-center">Reports</h1>
+        </div>
     </div>
 
 <form method="POST" action="{{ url_for('main.reports') }}">
```

Comment:
would be good to have some sort of formatting for the table header row, to show that it is the heading and below are the contents

Reply:
in a future version we really need to fix the look of this table, formatting of the results is much needed (but difficult to do without hardcoding levels of nestedness because of the variation in the result dict structure from hazenlib)

Reply:
Can you send me a screenshot? When I was last editing, I couldn't actually get any results into the table

Reply:
discussed in meeting, will add bolding later